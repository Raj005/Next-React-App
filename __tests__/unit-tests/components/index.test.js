import renderer from 'react-test-renderer';
import {
  Image,
  ResponsiveImageContainer,
  ModalImageComponent
} from '../../../app/ui/components/shared';

describe('Image component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Image />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('ResponsiveImageContainer component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<ResponsiveImageContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('ModalImageComponent component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<ModalImageComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
