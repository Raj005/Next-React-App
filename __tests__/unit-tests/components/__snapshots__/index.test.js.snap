// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component matches the snapshot 1`] = `
<img
  className="style__Image-l8s0xg-0 kpdFLk"
/>
`;

exports[`ModalImageComponent component matches the snapshot 1`] = `
<div>
  <img
    className="smallImage"
    onClick={[Function]}
    srcSet="undefined 375w, undefined 768w, undefined 1200w"
    style={
      Object {
        "cursor": "pointer",
        "maxHeight": "100%",
        "maxWidth": "100%",
      }
    }
  />
</div>
`;

exports[`ResponsiveImageContainer component matches the snapshot 1`] = `
<div
  className="style__ResponsiveImageWrapper-sc-11rzwhz-0 eRvRBP"
/>
`;
